libraries/hwlib/library/core/hwlib-i2c.hpp

```diff
@@ -494,7 +494,7 @@ class i2c_channel {
 /// - <a href="https://en.wikipedia.org/wiki/I2C">
 ///    I2C Bus</A> (wikipedia)
 ///
-class i2c_bus : protected i2c_primitives {
+class i2c_bus : public i2c_primitives {
```

Comment:
Why is the i2c bus changed in a usart pull request? It doesnt look like something is using the i2c_bus in the rest of the pull

Reply:
While testing with an oled I found that protected gave an error at this line when using i2c_bit_banged